Cargo.lock

```diff
@@ -1810,10 +1837,24 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "373b1a4c1338d9cd3d1fa53b3a11bdab5ab6bd80a20f7f7becd76953ae2be909"
 dependencies = [
  "arrayvec 0.7.2",
- "bitvec",
+ "bitvec 0.20.4",
  "byte-slice-cast",
  "impl-trait-for-tuples",
- "parity-scale-codec-derive",
+ "parity-scale-codec-derive 2.3.1",
+ "serde",
+]
+
+[[package]]
+name = "parity-scale-codec"
```

Comment:
We now have two versions of parity-scale-codec. Is this necessary?

I took a short look at the lock file, and it seems the old codec version is imported due to an old version of primitive-types, defined here: https://github.com/scs/substrate-api-client/blob/8a8f3059af1493e4eb9ccf114d98a4aea0dd1927/Cargo.toml#L21

Reply:
I will fix this. Good catch.